### PR TITLE
Carousel Scroll for Levels

### PR DESCRIPTION
**Pull request by jpark799**
[React-Multi-Carousel unfortunately does not allow you to start at a non-zero index](https://github.com/YIZHUANG/react-multi-carousel/issues/150)  so I had to reorder the array to have the current level be the first item. I had to make the carousel infinite so if you go to the left of your current level, you can still see the earlier levels.

### CODE REVIEW

File: packages/web-app/src/modules/seasons-views/pages/CurrentSeasonPage.tsx [hunk 1]
```diff
@@ -94,6 +99,9 @@ const _CurrentSeasonPage = ({
         }))
       : []
   }, [levels, nextLevel, currentLevelXP])
+
+  nextLevel >= 2 && levelCards.push(...levelCards.splice(0, nextLevel - 1))
```
Comment (dsarfati) [inline packages/web-app/src/modules/seasons-views/pages/CurrentSeasonPage.tsx:103]: What does this code do?? Is this moving the cards around?

**Review by dsarfati — CHANGES_REQUESTED** [review]
Are those the default arrows? Is there an issue using the same arrows we use for the reward list?

File: packages/web-app/src/modules/seasons-views/pages/CurrentSeasonPage.tsx (hunk 1 above)
Comment (jpark799) [reply packages/web-app/src/modules/seasons-views/pages/CurrentSeasonPage.tsx:103]: yeah, it's moving the current level to the front of the array and moving everything before the current array to the back of the array

**Comment by jpark799** [comment]
image

**Review by dsarfati — APPROVED** [review]

**Pull request merged**